### PR TITLE
Added empty `visit_For` function in the `SymbolTableVisitor` class

### PR DESCRIPTION
**Pull request by anutosh491**
Addresses the first part of https://github.com/lcompilers/lpython/issues/1287#issue-1444377844
```
j: i32
i: i32
for i in range(5):
    j = i
    print("j: ", j)
```
Something like this doesn't give us the expected output on main . It appears that the code is calling the `visit_stmt` function of the `SymbolTableVisitor`, but the `visit_For` function is only defined in the `BodyVisitor`. Hence an empty `visit_For` function in the `SymbolTableVisitor` class should do the job. Once this is done , we have the correct output
```
j:  0
j:  1
j:  2
j:  3
j:  4
```

### CODE REVIEW

**Review by czgdp1807 — APPROVED** [review]
Its okay to have it. I personally don't see any downsides. @certik Thoughts?

**Comment by certik** [comment]
I think that's fine. Should we add a test for this?

**Review by czgdp1807 — CHANGES_REQUESTED** [review]
Please add a test for this. See the existing pattern of integration_tests and add your own. Thanks.

**Comment by anutosh491** [comment]
> Please add a test for this. See the existing pattern of integration_tests and add your own. Thanks.

Sure , I'll do the needful soon !

File: integration_tests/loop_04.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from ltypes import i32
+j: i32
+i: i32
+for i in range(5):
+    j = i
+    print("j: ", j)
```
Comment (czgdp1807) [inline integration_tests/loop_04.py:6]: Register it in https://github.com/lcompilers/lpython/blob/main/integration_tests/CMakeLists.txt with `cpython`, `llvm`.

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

File: integration_tests/loop_04.py (hunk 1 above)
Comment (Thirumalai-Shaktivel) [reply integration_tests/loop_04.py:6]: @anutosh491 was there any issue?

**Review by czgdp1807 — APPROVED** [review]

**Comment by anutosh491** [comment]
Thanks for the reviews on this !

**Pull request merged**